### ISSUE TITLE
Multiple Swagger version with google_endpoints_service

### ISSUE THREAD

**Issue by gbergere** [terraform-providers/terraform-provider-google#4040]
Hello, I need to update my Google Cloud Endpoint definition to manage several API version.

It seems that I could do it with Google SDK (https://cloud.google.com/sdk/gcloud/reference/endpoints/services/). But I can't see anything on terraform-provider-google documentation https://www.terraform.io/docs/providers/google/r/endpoints_service.html.

I basically have 2 files openapi-v1.yml and openapi-v2.yml and I need to push both or them.

Would someone be able to help me about that or direct me to the proper documentation.

Many Thanks

**Comment by slevenick** [comment]
I don't believe the terraform resource currently supports this. It is built to send a single yaml file as the endpoint definition. It would certainly make sense to support a list of yaml configs in the future

**Comment by gbergere** [comment]
Thanks for your answer @slevenick

**Comment by mitj04** [comment]
I agree with this [comment](https://github.com/hashicorp/terraform-provider-google/issues/4040#issuecomment-512598760). The openapi config field currently only supports a single file source because it is a type of `TypeString`. We might modify the source's type to `TypeList` and handle it differently to accommodate multiple file sources, but as I noted, this might necessitate changing the schema. Thus, this might be a `breaking-change`.

**Comment by mitj04** [comment]
b/262749412
